### PR TITLE
Add empty state when ID in url isn't in the DB

### PR DESCRIPTION
**Pull request by matoval**
# Description

Add empty state to be shown if ID in url doesn't match an image in the organizations database 

Fixes # (issue)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: src/Routes/ImageManagerDetail/DetailsHeader.js [hunk 1]
```diff
@@ -65,90 +65,100 @@ const DetailsHead = ({ imageData, imageVersion, openUpdateWizard }) => {
 
   return (
     <>
-      <Breadcrumb>
-        <BreadcrumbItem>
-          <Link to={paths['manage-images']}>Manage Images</Link>
-        </BreadcrumbItem>
-        {imageVersion ? (
+      {data?.image_set && !data?.image_set?.Account ? (
```
Comment (boaz0) [inline src/Routes/ImageManagerDetail/DetailsHeader.js:68]: how about an helper function to check this?

```js
const isAccountMissing = (imageSet) => imageSet && !imageSet?.Account
```

File: src/Routes/ImageManagerDetail/ImageDetailTabs.js [hunk 2]
```diff
@@ -55,47 +57,67 @@ const ImageDetailTabs = ({
...
-                <Skeleton width="75px" />
-              </TabTitleText>
-            }
-          ></Tab>
-        ) : imageVersion ? (
-          <Tab
-            eventKey={tabs.packages}
-            title={<TabTitleText>Packages</TabTitleText>}
+    <>
+      {imageData?.data?.Data?.image_set &&
```
Comment (boaz0) [inline src/Routes/ImageManagerDetail/ImageDetailTabs.js:61]: use the helper function mentioned above (`isAccountMissing`):

```js
isAccountMissing(imageData?.data?.Data?.image_set)
```

File: src/Routes/ImageManagerDetail/ImageDetailTabs.js [hunk 3]
```diff
@@ -55,47 +57,67 @@ const ImageDetailTabs = ({
...
+              <ImageDetailTab
+                imageData={imageData}
+                imageVersion={imageVersion}
+              />
+            </Tab>
+            {isLoading ? (
+              <Tab
+                title={
+                  <TabTitleText>
+                    {' '}
```
Comment (boaz0) [inline src/Routes/ImageManagerDetail/ImageDetailTabs.js:94]: is this `{' '}` necessary or was added because of `eslint`?

**Review by boaz0 — COMMENTED** [review]
LGTM :+1: just some nits.

File: src/Routes/ImageManagerDetail/DetailsHeader.js (hunk 1 above)
Comment (matoval) [reply src/Routes/ImageManagerDetail/DetailsHeader.js]: Great suggestion, Thanks @boaz0!

**Review by adeliaferreira — APPROVED** [review]
lgtm

**Pull request merged**